src/main/java/pl/jkap/sozzt/contract/exception/MyNotYetImplementedException.java

```diff
@@ -0,0 +1,13 @@
+package pl.jkap.sozzt.contract.exception;
+
+import org.hibernate.cfg.NotYetImplementedException;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NOT_IMPLEMENTED)
+public class MyNotYetImplementedException extends NotYetImplementedException {
+
+    public MyNotYetImplementedException() {
+        super();
+    }
```

Comment:
If you are not giving any additional info, why not use `NotYetImplementedException` ?